docs/development/virtual_environments.rst

```diff
@@ -1,14 +1,25 @@
 Virtual Environments: Servers
 =============================
 
-There are two predefined virtual environments in the Vagrantfile:
+SecurerDrop is a multi-server system, and  you may need the full server
+stack available in order to develop and test some features. To make this easier, 
+the project includes a Vagrantfile that can be used to create two predefined 
+virtual environments:
 
-* :ref:`Staging <staging_vms>`
-* :ref:`Production <production_vms>`
+* :ref:`Vagrant: staging <staging_vms>`
+* :ref:`Vagrant: production <production_vms>`
```

Comment:
Would prefer to keep the generalized env names, omitting specific technologies (especially since we're moving away from direct use of Vagrant).

Reply:
:heavy_check_mark: